Update delete-site-collection.md (#1057)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/delete-site-collection.md

    
You should notify the site admins and any subsite owners before you delete a site so they can move their data to another location, and also tell users when the sites will be deleted. 

> Important: It is not recommended to delete the root site collection. If you delete the root site collection, all SharePoint sites in your organization will be inaccessible until you restore the site or create a new site at <Root URL>

Reviewer: ```suggestion
Important: Deleting the root site collection is not recommended. If you delete the root site collection, all SharePoint sites in your organization will be inaccessible until you restore the site or create a new site at <Root URL>.
```
